OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -87,6 +87,20 @@ public string GetCursor(World world, CPos xy, MouseInput mi)
 			return cursorName ?? (useSelect ? "select" : "default");
 		}
 
+		// Used for classic mouse orders, determines whether or not action at xy is move or select
+		public static bool DetermineAmbiguity(World world, int2 xy, MouseInput mi)
```

Comment:
This isn't a very descriptive name.  How about something like `ShouldOverrideSelection`?